datavault-broker/src/main/java/org/datavaultplatform/broker/controllers/admin/AdminController.java

```diff
@@ -143,8 +143,10 @@ public void setSender(Sender sender) {
             Vault vault = vaultsService.getVault(depositInfo.getVaultID());
             depositInfo.setVaultName(vault.getName());
             User vaultOwner = vault.getUser();
-            depositInfo.setVaultOwnerID(vaultOwner.getID());
-            depositInfo.setVaultOwnerName(vaultOwner.getFirstname() + " " + vaultOwner.getLastname());
+            if (vaultOwner != null) {
+                depositInfo.setVaultOwnerID(vaultOwner.getID());
```

Comment:
Does the depositInfo object works fine with null values in the ownerid/ownername fields? The DB has nullable field and there aren't other places where code accesses that field without null check?

Reply:
Yeah it should work fine. The only places that use these fields are in the freemarker templates and I didn't notice any errors when I was testing this